src/mainwindow.cpp

```diff
@@ -1480,8 +1485,8 @@ void MainWindow::onDotsButtonClicked()
     }
 
     // Enable or Disable markdown
-    QString markDownLabel = m_noteEditorLogic->markdownEnabled() ? tr("Disable Markdown")
-                                                                 : tr("Enable Markdown");
+    QString markDownLabel = m_noteEditorLogic->markdownEnabled() ? tr("&Disable Markdown")
+                                                                 : tr("&Enable Markdown");
```

Comment:
I think it might be better to put the mnemonic on M of Markdown. In general it should better to not need a separate key for enable/disable or show/hide.